coreos/master-config.yaml

```diff
@@ -22,13 +22,15 @@ write_files:
     permissions: 0755
     content: |
       #! /usr/bin/bash
+      # network not always up, ping to wait for internet connection
+      ping google.com -c 5
```

Comment:
Better.

What if we do this:
```
while ! ping -c 1 $KUBERNETES_LOC; do :; done
```

We are expected to have connectivity to the KUBERNETES_LOC.  Not necessarily true of Google.